app/models/manageiq/providers/embedded_ansible/seeding.rb

```diff
@@ -19,6 +19,43 @@ def seed
       ).find_or_create_by!(
         :name => "#{Vmdb::Appliance.PRODUCT_NAME} Default Credential"
       )
+
+      create_local_playbook_repo
+      local_repo = ManageIQ::Providers::EmbeddedAnsible::AutomationManager::ConfigurationScriptSource.find_or_create_by!(
+        :name       => "#{Vmdb::Appliance.PRODUCT_NAME} Default Project",
+        :manager_id => manager.id
+      )
+      local_repo.raw_update_in_provider(
+        :scm_type             => "git",
+        :scm_url              => "file://#{local_playbook_repo_dir}",
+        :scm_update_on_launch => false
+      )
+    end
+
+    private
+
+    def create_local_playbook_repo
+      Ansible::Content.consolidate_plugin_playbooks
```

Comment:
I feel like maybe this should be done at build time...or perhaps we can do it at seed time for devs, but if, say, the directory already exists because we did it at build time, then this is a no-op.  I'm not sure.

It feels like this would be expensive to do this at seed time, which is already so slow.  I'm not sure.

Reply:
I'm fine with either. We did this at runtime with the previous implementation out of necessity (ansible tower had to be running for us to add the repo).

We can put this bit into `Ansible::Content` and make a rake task out of it to be called at build time, but obviously the creation of the configuration script source still has to be here.

Reply:
Doing this at build time would make the code a bit messy and make this much harder for devs who want to run it. I also don't think it will be expensive.